rules/php74/tests/Rector/FuncCall/ArraySpreadInsteadOfArrayMergeRector/Fixture/iterator_to_array.php.inc

```diff
@@ -16,11 +16,11 @@ class IteratorToArray
 
 namespace Rector\Php74\Tests\Rector\FuncCall\ArraySpreadInsteadOfArrayMergeRector\Fixture;
 
-class IteratorToArray
+final class IteratorToArray
 {
     public function run($items)
     {
-        $values = [...$items];
+        $values = iterator_to_array($items);
     }
 }
```

Comment:
The second code block start with `-----` can be removed entirely if code change is skipped. You may can also rename the fixture to `skip_iterator_to_array.php.inc`.

Reply:
Thanks, fixed!